CONTRIBUTING.md

```diff
@@ -118,3 +118,8 @@ and every commit is able to be built and passes all checks performed by CI.
 
 The Aptos SDK uses [GitHub issues](https://github.com/aptos-labs/aptos-ts-sdk/issues) to track bugs. Please include 
 necessary information and instructions to reproduce your issue.
+
+## Releasing a new version
+
+Simply update the version in `package.json` and `pnpm update-version` will take care of the rest. This will create a new
```

Comment:
lets also mention checking out to a new branch before updating/running the command